stl.go

```diff
@@ -595,18 +638,46 @@ func newTTIBlock(i *Item, idx int) (t *ttiBlock) {
 		subtitleNumber:       idx,
 		timecodeIn:           i.StartAt,
 		timecodeOut:          i.EndAt,
-		verticalPosition:     20,
+		verticalPosition:     verticalPositionFromStyle(i.InlineStyle),
 	}
 
 	// Add text
 	var lines []string
 	for _, l := range i.Lines {
-		lines = append(lines, l.String())
+		var lineItems []string
+		for _, li := range l.Items {
+			lineItems = append(lineItems, li.STLString())
+		}
+		lines = append(lines, strings.Join(lineItems, " "))
 	}
-	t.text = []byte(strings.Join(lines, "\n"))
+	t.text = []byte(strings.Join(lines, string(rune(stlLineSeparator))))
 	return
 }
 
+func verticalPositionFromStyle(sa *StyleAttributes) int {
```

Comment:
Could you prefix this method with `stl` ? (`stlVerticalPositionFromStyle`)

Reply:
Done.